src/sonic-yang-models/yang-models/sonic-kubernetes_master.yang

```diff
@@ -0,0 +1,60 @@
+module sonic-kubernetes_master {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-kubernetes_master";
+    prefix kubernetes_master;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description "KUBERNETES_MASTER YANG Module for SONiC OS";
+    
+    revision 2022-10-09 {
+        description "First Revision";
+    }
+
+    container sonic-kubernetes_master {
+
+        container KUBERNETES_MASTER {
+
+            description "KUBERNETES_MASTER part of config_db.json";
+
+            container SERVER {
+
+                leaf ip {
+                    description "Kubernetes api server endpoint";
+                    type string;
+                }
+
+                leaf port {
+                    description "Kubernetes api server port";
+                    type inet:port-number;
+                }
+
+                leaf disable {
+                    description "This configuration identicates whether disable kubernetes";
+                    type stypes:boolean_type;
+                    default "false";
+                }
+
+                leaf insecure {
+                    description "This configuration identicates it will download kubernetes
+                                CA by http other than https";
+                    type stypes:boolean_type;
+                    default "true";
```

Comment:
This should be false too.

Reply:
Currently sonic join logic is to download k8a default ca.crt and combine it with local ame certificate, we get the kubeconfig credential. The default ca.crt is stored in k8s configmap, we download it from apiserver endpoint, and sonic impossibly have the default ca.crt, but if sonic want to establish secure connection with apiserver, sonic needs k8s default ca.crt. So, the downloading k8s default ca request can only be insecure currently. So, I set the default value "true".